.github/workflows/test.yaml

```diff
@@ -126,12 +126,10 @@ jobs:
   parametric:
     needs:
       - build-artifacts
-    uses: DataDog/system-tests/.github/workflows/run-parametric.yml@a675fcc0b1d582485c57c4ab0c30cf2f9eb217ea # igor/rust-the-germ
+    uses: DataDog/system-tests/.github/workflows/run-parametric.yml@main
```

Comment:
I think that we should have a fixed hash here so we can move it and include new tests as we see fit. Having a moving parametric test target would also make supporting multiple versions hard (yes, we're not there yet).